recipe/meta.yaml

```diff
@@ -1,15 +1,18 @@
 {% set name = "y-py" %}
-{% set version = "0.5.9" %}
+{% set version = "0.6.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/y/y_py/y_py-{{ version }}.tar.gz
-  sha256: 50cfa0532bcee27edb8c64743b49570e28bb76a00cd384ead1d84b6f052d9368
+  sha256: 4757a82a50406a0b3a333aa0122019a331bd6f16e49fed67dca423f928b3fd4d
 
 build:
+  run_exports: # [osx and x86_64]
+    strong_constrains: # [osx and x86_64]
+      - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.12") }}  # [osx and x86_64]
```

Comment:
Do we really need this? You'll get these for free if you use your latest builds for rust and rust-activation if I'm not mistaken.

Reply:
I was just about to ask about this as well